commands/logging-service/logs/add.go

```diff
@@ -0,0 +1,124 @@
+package logs
+
+import (
+	"context"
+	"strconv"
+	"strings"
+
+	"github.com/ionos-cloud/ionosctl/v6/commands/logging-service/completer"
+	"github.com/ionos-cloud/ionosctl/v6/internal/client"
+	"github.com/ionos-cloud/ionosctl/v6/internal/constants"
+	"github.com/ionos-cloud/ionosctl/v6/internal/core"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/tabheaders"
+	ionoscloud "github.com/ionos-cloud/sdk-go-logging"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func LogsAddCmd() *core.Command {
+	cmd := core.NewCommand(
+		context.Background(), nil, core.CommandBuilder{
+			Namespace: "logging-service",
+			Resource:  "logs",
+			Verb:      "add",
+			ShortDesc: "Add a log to a logging pipeline",
+			Example: `ionosctl logging-service logs add --pipeline-id ID --log-tag TAG --log-source SOURCE --log
+-protocol PROTOCOL`,
+			PreCmdRun: preRunAddCmd,
+			CmdRun:    runAddCmd,
+		},
+	)
+	cmd.Command.Flags().StringSlice(constants.ArgCols, defaultCols, tabheaders.ColsMessage(defaultCols))
+	cmd.AddStringFlag(
+		constants.FlagLoggingPipelineId, constants.FlagIdShort, "",
+		"The ID of the logging pipeline", core.RequiredFlagOption(),
+	)
+	_ = cmd.Command.RegisterFlagCompletionFunc(
+		constants.FlagLoggingPipelineId,
+		func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+			return completer.LoggingServicePipelineIds(), cobra.ShellCompDirectiveNoFileComp
+		},
+	)
+	cmd.AddStringFlag(
+		constants.FlagLoggingPipelineLogTag, "", "", "Sets the tag for the pipeline log", core.RequiredFlagOption(),
+	)
+	cmd.AddSetFlag(
+		constants.FlagLoggingPipelineLogSource, "", "", []string{"docker", "systemd", "generic", "kubernetes"},
```

Comment:
this is minor, but since these values are used in multiple places across the logging-service package, we could create some constants for them.